README.markdown

```diff
@@ -278,8 +278,11 @@ It is possible to specify the useraccount (and password) that the service will r
        <domain>YOURDOMAIN</domain>
        <user>useraccount</user>
        <password>Pa55w0rd</password>
+       <allowserviceaccountlogonright>true</allowserviceaccountlogonright>
```

Comment:
This is all lower case is it not?  Doesn't really matter since I will be renaming this anyways - just need to understand your comment.